main.go

```diff
@@ -6,22 +6,18 @@ package main
 import (
 	"fmt"
 	"os"
-	"packer-plugin-scaffolding/builder/scaffolding"
-	scaffoldingData "packer-plugin-scaffolding/datasource/scaffolding"
-	scaffoldingPP "packer-plugin-scaffolding/post-processor/scaffolding"
-	scaffoldingProv "packer-plugin-scaffolding/provisioner/scaffolding"
-	scaffoldingVersion "packer-plugin-scaffolding/version"
+
+	"github.com/packer-plugin-kubearmor/provisioner/kubearmor"
+	Version "github.com/packer-plugin-kubearmor/version"
```

Comment:
```suggestion
	version "github.com/packer-plugin-kubearmor/version"
```

as version is not exported